jina/peapods/jinad.py

```diff
@@ -0,0 +1,193 @@
+import uuid
+from pathlib import Path
```

Comment:
can we have a submodule inside `peapods` called `jinad` and there have `JinadAPI` , `PeaAPI` and `PodAPI`?

Reply:
This is just for api invocations. There's no need to add jinad submodules, as the api creation can be completely separate

Reply:
well, but is a different concept, to me it feels like deserving a new module

Reply:
The usage of jinad apis invocations won't have much more additions like we have in jina-hub. Also, we do `hubapi` invocations in hubio. I don't see a reason for these to be submodules.

Reply:
For me is a mattter of clarity, but as u want